proto/tenancy_config/ota_config.proto

```diff
@@ -0,0 +1,12 @@
+﻿syntax = "proto3";
+
+package holms.types.tenancy_config;
+option csharp_namespace = "HOLMS.Types.TenancyConfig";
+option go_package = "tenancyconfig";
+
+message OTAConfig {
+	string provider_i_d = 1;
+	string provider_user_name = 2;
+	string provider_password = 3;
+	string hotel_i_d = 4;
```

Comment:
Yeah. This is mapping to a setting in the connection to the channel manager. The "hotel ID" is equivalent to a tenancy in our system right now. The channel rush account may be shared across multiple tenancies.